### PR TITLE
Feat java runtime

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

**Comment by christyjacob4** [comment]
Looks great 👌 
Can you please include these new runtimes in https://github.com/appwrite/runtimes/pulls

**Comment by christyjacob4** [comment]
We also need to add the build steps this file 
https://github.com/open-runtimes/open-runtimes/blob/main/build.sh

File: runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeResponse.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    public RuntimeResponse send(String data, int statusCode) {
+        this.data = data;
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    public static RuntimeResponse error(Throwable error) {
+        return new RuntimeResponse("{\"message\": \"" + error.getMessage() + "\"}", 500);
```
Comment (christyjacob4) [inline runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeResponse.java:46]: Same here, we no longer return a JSON string for the error 
https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L45

File: runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeRequest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ) {
+        this.payload = payload;
+        this.headers = headers;
+        this.env = env;
+    }
+
+    public RuntimeRequest(Req request) {
+        Map<String, Object> data = request.data();
+        if (data.containsKey("payload")) {
+            this.payload = (String) data.get("payload");
```
Comment (christyjacob4) [inline runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeRequest.java:25]: The default payload must be an empty string as you can see here. So please verify that once

https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L21

These changes are currently in the 0.14 branch

File: runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeResponse.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    public static RuntimeResponse unauthorized() {
+        return new RuntimeResponse("{\"message\": \"Unauthorized\"}", 401);
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeResponse.java:50]: Please ensure you double check for the other runtimes as well

**Comment by abnegate** [comment]
@christyjacob4 Updated

**Pull request merged**